CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Fixed `crate_authors!` macro usage, this macro can't return static string in new clap version. (#370)
 - Fixed mistake in description of the height getter in the `ServiceContext`. (#366)
+- Fixed `api` listeners config was override by runtime flag. (377)
```

Comment:
And did the releases have this problem? In my opinion this record is redundant.

Reply:
I agree. @vldm Could you remove it?